partner_event/__manifest__.py

```diff
@@ -8,7 +8,7 @@
 
 {
     'name': 'Link partner to events',
-    'version': '10.0.1.0.0',
+    'version': '10.0.1.0.1',
```

Comment:
Hmm this should be `10.0.2.0.0`, since it means running a DB migration and is not completely backwards-compatible, right? 🤔